pravega-cli/src/test/java/io/pravega/tools/pravegacli/SecureControllerCommandsTest.java

```diff
@@ -0,0 +1,170 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
```

Comment:
Remove the year from the Copyright statement.

Reply:
Removed the year from the statement